Formula/m/mumps.rb

```diff
@@ -0,0 +1,56 @@
+class Mumps < Formula
+  desc "MUltifrontal Massively Parallel sparse direct Solver"
+  homepage "https://mumps-solver.org/"
+  url "https://mumps-solver.org/MUMPS_5.6.2.tar.gz"
+  sha256 "13a2c1aff2bd1aa92fe84b7b35d88f43434019963ca09ef7e8c90821a8f1d59a"
+  license "CECILL-C"
+
+  # Core dependencies
+  depends_on "gcc"
+  depends_on "openblas"
+  fails_with :clang
+
+  def install
+    makefile = "Makefile.G95.SEQ"
+    cp "Make.inc/" + makefile, "Makefile.inc"
+    inreplace "Makefile.inc", "-soname", "-install_name" unless OS.linux?
+    inreplace "Makefile.inc", ".so", ".dylib" unless OS.linux?
+    inreplace "Makefile.inc", "# RPATH_OPT = -Wl,-rpath,/path/to/MUMPS_x.y.z/lib/", "RPATH_OPT = -Wl,-rpath,#{lib}"
```

Comment:
Have these issues been reported/fixed upstream?

Reply:
I am discussing ln:16 and ln:17 with the MUMPS developers, ln:18 is not an issue, it's an expected and documented part of the MUMPS build workflow if you want to build w/ RPATHS.

Reply:
> it's an expected and documented part of the MUMPS build workflow if you want to build w/ RPATHS.

Can you link to the documentation then here please?

```suggestion
    # Required to build with RPATH: 
    # https://some.li/nk
    inreplace "Makefile.inc", "-soname", "-install_name" unless OS.linux?
    inreplace "Makefile.inc", ".so", ".dylib" unless OS.linux?
    inreplace "Makefile.inc", "# RPATH_OPT = -Wl,-rpath,/path/to/MUMPS_x.y.z/lib/", "RPATH_OPT = -Wl,-rpath,#{lib}"
```